refactor: remove `chatInputCommand` options (#181)
sapphiredev/website: docs/Guide/getting-started/creating-a-basic-app-command.mdx

title: Creating a basic slash command
---

Creating a basic slash command is very similar to creating a basic command. To create a slash command you will have to

> call the `registerApplicationCommands` method of the command class.

Reviewer: ```suggestion
define the `registerApplicationCommands` method in the command class.
```

export class PingCommand extends Command {
  public constructor(context: Command.Context, options: Command.Options) {
    super(context, { ...options });
  }

  public override registerApplicationCommands(registry: ChatInputCommand.Registry) {
    registry.registerChatInputCommand((builder) =>

>       builder.setName('ping').setDescription('Ping bot to see if it is alive')

Reviewer: Maybe we can also show an example of reusing this.name/this.description?
